documentation/asciidoc/computers/raspberry-pi/eeprom-bootloader.adoc

```diff
@@ -481,6 +521,14 @@ PARTITION=2
 
 Default: 0
 
+[[PARTITION_WALK]]
+==== `PARTITION_WALK`
+This property is designed to improve the reliability of `A/B` boot schemes using `autoboot.txt` by searching for bootable partitions if the specified partition does not appear to be bootable. If `PARTITION_WALK=1` and the requested partition is not bootable and does not have a valid `autoboot.txt` then the bootloader will check each partition in turn (up to 8 and wrapping to 0) to see if it is bootable (contains `start4.elf` on a Pi4 or `config.txt` and a suitable device-tree on Pi 5 or newer).
```

Comment:
nitpick: IMHO there ought to be a comma after Pi4 (as this more clearly delineates the Pi 4 and Pi 5 cases), and "Pi4" ought to be "Pi 4" :slightly_smiling_face:

Reply:
Go on then, you can have the comma